### DISCUSSION TITLE
Failed to create admin user

### DISCUSSION THREAD

**Discussion by oitkenmen** [Budibase/budibase#7699] [Help] [answered]
Hello, there is an error after installation on AWS kubernetes: `Failed to create admin user`. I tried reinstalling the app 5 or 6 times now, but it still shows the same error. It looks like it started since v1.3. How to install the previous version of Budibase?

**Comment by AiCaptain** [comment]
Same issue on our selfhosted k8s cluster. 

Some background infos:

I used the helm chart to uninstall and (re-)install budibase, after an upgrade from 1.2.52 to 1.3.11 via helm produced an "Error initialising app:" error for every app. So I uninstalled the release and also deleted the pvs to start from scratch.

After the fresh install I wonder about starting with the organization screen although the multitenancy flag is explicitly set to "0" in my values.yaml. Ultimately, I also end up getting the "Unable to retrieve user list" message when i try to create a super admin user.

**Reply by oitkenmen** [reply]
Hi, do you know how to install v1.2.52 using helm?

**Reply by AiCaptain** [reply]
Yes, you have to set
globals:
  appVersion: "v1.2.52"
in your own override-values.yaml or via cli with
helm install ... --set globals.appVersion="v1.2.52"

**Reply by AiCaptain** [reply]
Btw, the v1.2.52 works as expected, all v1.3.X version have the same problems in our environment ...

**Reply by oitkenmen** [reply]
Thank you so much!

**Comment by bilus** [comment]
We are having the same issue here. :( I can share more configuration details if necessary.

**Comment by ellakk** [comment]
Yeah, we got the same issue too, running the latest version of the helm chart

**Comment by AntorFr** [comment]
Same on my side trying with version v2.1.28 .. any idea to solve this issue ?

**Reply by AntorFr** [reply]
I try with version v1.2.52 and it works, but all recent version seems to have the same issues, you cannot instanciated them (at least with a k3s using helm) :(

**Comment by oitkenmen** [comment, marked as answer]
The solution is to keep the field "Organization" empty. Basically, to hit the "Enter" button 2 times. And you can see that the field shows "default" value. Then you can create an admin.

**Reply by AntorFr** [reply]
Woh ! ... fighting on it since 3 days, and this solve the issue ! Many thanks @berkimbayev for the tips.
Would be good to fix this for good, but at least there is a workaround.
